core/forms.py

```diff
@@ -105,6 +117,80 @@ def __init__(self, *args, **kwargs):
             visible.field.widget.attrs["class"] = "form-button-style py-2"
 
 
+class HackerSettingForm(forms.ModelForm):
+    country = forms.CharField(max_length=255)
+    city = forms.CharField(max_length=255)
+    school = forms.CharField(max_length=255, required=False)
+    education = forms.ChoiceField(
+        choices=HACKATHON_EDUCATION_CHOICES, widget=forms.Select
+    )
+    username = forms.CharField(max_length=255)
+    email = forms.CharField(max_length=255)
```

Comment:
Not as simple as just email. that needs to verified separately 
https://docs.allauth.org/en/dev/account/configuration.html